src/lmdb.ml

```diff
@@ -217,11 +216,15 @@ module Flags = struct
   let create     = mdb_CREATE
 end
 
+type buffer =
+  (char, Bigarray.int8_unsigned_elt, Bigarray.c_layout) Bigarray.Array1.t
+
 module Values = struct
 
   module Flags = Flags
 
-  type db_val = (mdb_val, [ `Struct ]) structured ptr
+  type db_val = buffer
+  let alloc_buffer = Bigarray.(Array1.create Char C_layout)
```

Comment:
I didn't see imediately that this was an allocation *function*. Could you eta-expand and rename it ?

Reply:
I eta-epanded it, but I don't see what's wrong with the name. Please suggest an acceptable name.
```suggestion
  let alloc_buffer len = Bigarray.(Array1.create Char C_layout len)
```

Reply:
Maybe `allocate_buffer` ? alloc for me is short for "allocation", which is a noun which suggests a value, allocate is a verb, whch suggests a function.

Reply:
In any case, that function is gone for now!